app/assets/javascripts/embed.js

```diff
@@ -0,0 +1,38 @@
+import Embed from 'flat-embed';
```

Comment:
So adding this file to our javascript assets folder solves the autoloading problem, Embed that I'm importing is coming from the npm package manager

Reply:
So after testing in local host a little more, this file basically makes it so that javascript on the page, only has to be loaded once, instead of every time you visit the page. so the problem still persists when a user initially visits the site but once they load the editor once, it stops, so this is closer to the answer